src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1656,68 +1207,21 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         const auto point = args.GetCurrentPoint(*this);
         const auto props = point.Properties();
-        const TerminalInput::MouseButtonState state{ props.IsLeftButtonPressed(), props.IsMiddleButtonPressed(), props.IsRightButtonPressed() };
-        auto result = _DoMouseWheel(point.Position(),
-                                    ControlKeyStates{ args.KeyModifiers() },
-                                    point.Properties().MouseWheelDelta(),
-                                    state);
+        const TerminalInput::MouseButtonState state{ props.IsLeftButtonPressed(),

```

Comment:
why's this different from the other TerminalThingToMouseButtons thing?

Reply:
Probably because it was always different and I forgot to dedupe this one 😋